File: AppGenerateFiles/security/Model/Tbl_Profile.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Tbl_Profile : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? Id_Perfil { get; set; }
       public string? Nombres { get; set; }
       public string? Apellidos { get; set; }
       public DateTime? FechaNac { get; set; }
       public int? IdUser { get; set; }
       public string? Sexo { get; set; }
       public string? Foto { get; set; }
       public string? DNI { get; set; }
       public string? Correo_institucional { get; set; }
       public int? Id_Pais_Origen { get; set; }
       public string? Estado { get; set; }
       [ManyToOne(TableName = "Cat_Paises", KeyColumn = "Id_Pais", ForeignKeyColumn = "Id_Pais_Origen")]
       public Cat_Paises? Cat_Paises { get; set; }
       [ManyToOne(TableName = "Security_Users", KeyColumn = "Id_User", ForeignKeyColumn = "IdUser")]
       public Security_Users? Security_Users { get; set; }
       [OneToMany(TableName = "CaseTable_Agenda", KeyColumn = "Id_Perfil", ForeignKeyColumn = "Id_Perfil")]
       public List<CaseTable_Agenda>? CaseTable_Agenda { get; set; }
       [OneToMany(TableName = "CaseTable_Case", KeyColumn = "Id_Perfil", ForeignKeyColumn = "Id_Perfil")]
       public List<CaseTable_Case>? CaseTable_Case { get; set; }
       [OneToMany(TableName = "CaseTable_Dependencias_Usuarios", KeyColumn = "Id_Perfil", ForeignKeyColumn = "Id_Perfil")]
       public List<CaseTable_Dependencias_Usuarios>? CaseTable_Dependencias_Usuarios { get; set; }
       [OneToMany(TableName = "CaseTable_Participantes", KeyColumn = "Id_Perfil", ForeignKeyColumn = "Id_Perfil")]
       public List<CaseTable_Participantes>? CaseTable_Participantes { get; set; }
       [OneToMany(TableName = "Tbl_Profile_CasosAsignados", KeyColumn = "Id_Perfil", ForeignKeyColumn = "Id_Perfil")]
       public List<Tbl_Profile_CasosAsignados>? Tbl_Profile_CasosAsignados { get; set; }
       [OneToMany(TableName = "Tbl_Servicios_Profile", KeyColumn = "Id_Perfil", ForeignKeyColumn = "Id_Perfil")]
       public List<Tbl_Servicios_Profile>? Tbl_Servicios_Profile { get; set; }
   }
}

File: AppGenerateFiles/security/FrontModel/Tbl_Profile.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
import { WAjaxTools } from "../../WDevCore/WModules/WComponentsTools.js";
//@ts-ignore
import { ModelProperty } from "../../WDevCore/WModules/CommonModel.js";
import { Cat_Paises }  from './Cat_Paises.js'
import { Security_Users }  from './Security_Users.js'
import { CaseTable_Agenda }  from './CaseTable_Agenda.js'
import { CaseTable_Case }  from './CaseTable_Case.js'
import { CaseTable_Dependencias_Usuarios }  from './CaseTable_Dependencias_Usuarios.js'
import { CaseTable_Participantes }  from './CaseTable_Participantes.js'
import { Tbl_Profile_CasosAsignados }  from './Tbl_Profile_CasosAsignados.js'
import { Tbl_Servicios_Profile }  from './Tbl_Servicios_Profile.js'
class Tbl_Profile extends EntityClass {
   constructor(props) {
       super(props, 'EntitySecurity');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ Id_Perfil;
   /**@type {String}*/ Nombres;
   /**@type {String}*/ Apellidos;
   /**@type {Date}*/ FechaNac;
   /**@type {String}*/ Sexo;
   /**@type {String}*/ Foto;
   /**@type {String}*/ DNI;
   /**@type {String}*/ Correo_institucional;
   /**@type {String}*/ Estado;
   /**@type {Cat_Paises} ManyToOne*/ Cat_Paises;
   /**@type {Security_Users} ManyToOne*/ Security_Users;
   /**@type {Array<CaseTable_Agenda>} OneToMany*/ CaseTable_Agenda;
   /**@type {Array<CaseTable_Case>} OneToMany*/ CaseTable_Case;
   /**@type {Array<CaseTable_Dependencias_Usuarios>} OneToMany*/ CaseTable_Dependencias_Usuarios;
   /**@type {Array<CaseTable_Participantes>} OneToMany*/ CaseTable_Participantes;
   /**@type {Array<Tbl_Profile_CasosAsignados>} OneToMany*/ Tbl_Profile_CasosAsignados;
   /**@type {Array<Tbl_Servicios_Profile>} OneToMany*/ Tbl_Servicios_Profile;
}
export { Tbl_Profile }
